src/darwin/Framework/CHIP.xcodeproj/project.pbxproj

```diff
@@ -618,6 +654,8 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				GCC_PREPROCESSOR_DEFINITIONS = (
 					CHIP_HAVE_CONFIG_H,
+					"CHIP_IM_MAX_NUM_READ_CLIENT=8",
```

Comment:
This should be in the platform config, not the Xcode file.

Reply:
Removed them because ToT doesn't use those any more.